### PR TITLE
Added TFBartForSequenceClassification

### LINKED ISSUE #19653: TFBartForSequenceClassification
The Tensorflow version of `BartForSequenceClassification` seems to be missing.  I've been putting a port together for another use case.  Any interest in adding it to the repo?

### PR DESCRIPTION
**Pull request by IMvision12**
# What does this PR do?

Fixes: #19653

### CODE REVIEW

**Review by sgugger — APPROVED** [review]
Thanks for your PR! Could you make sure your new model is also tested by adding it in `tests/models/bart/test_modeling_tf_bart.py`?

**Comment by IMvision12** [comment]
@sgugger I have tested locally, not sure why tests are failing

**Comment by sgugger** [comment]
@IMvision12 the common tests do not pass for the model you have added, you can run them with `pytest tests/models/bart/test_modeling_tf_bart.py`.

Also the equivalence test PyTorch/TensorFlow does not pas either.

**Comment by IMvision12** [comment]
@sgugger I've looked at some SequenceClassification models and the tests that have been written but none of them have TFSeq2SeqSequenceClassification or tests for it, so I'm not sure how to add a test for that.

File: src/transformers/models/bart/modeling_tf_bart.py [hunk 1]
```diff
@@ -1465,3 +1468,120 @@ def _reorder_cache(past, beam_idx):
...
+            output = (logits,) + outputs[1:]
+            return ((loss,) + output) if loss is not None else output
+
+        return TFSeq2SeqSequenceClassifierOutput(
+            loss=loss,
+            logits=logits,
+            past_key_values=outputs.past_key_values,
+            decoder_hidden_states=outputs.decoder_hidden_states,
+            decoder_attentions=outputs.decoder_attentions,
+            encoder_last_hidden_state=outputs.encoder_last_hidden_state,
```
Comment (ydshieh) [inline src/transformers/models/bart/modeling_tf_bart.py:1567]: There is a missing `cross_attentions=outputs.cross_attentions,` here. We should fix `TFSeq2SeqSequenceClassifierOutput` to have the same attributes as `Seq2SeqSequenceClassifierOutput`.

Could you try it, @IMvision12 

For other issues, let's check next week

**Comment by IMvision12** [comment]
@ydshieh Still the tests are failing

**Comment by IMvision12** [comment]
This issue has been solved in this PR here #20570 so i am closing this PR

**Pull request closed without merging**